SnookR/sublist/templates/sublist/sublist.html

```diff
@@ -4,9 +4,8 @@
 <head>
   <meta charset="utf-8">
 
-  <title>McAnulty and Barry's Sublist</title>
-  <meta name="description" content="McAnulty and Barry's Sublist">
-  <meta name="author" content="Evan Smelser">
+  <title>{{ sublist.name }}'s Sublist</title>
+  <meta name="description" content="{{ sublist.name }}'s Sublist">
```

Comment:
@eSmelser bootstrap is great and I've used it for all my other projects.  Luckily bootstrap won't actually change any functionality, just presentation.  So we can build all the buttons and pages and links and everything, and then use bootstrap to make it all look pretty and professional.  Do you have an websites in mind that have a similar style or format as to what you are looking for?  We could start off by making a clone, more or less, and then modifying it.